app/(dashboard)/notes/page.tsx

```diff
@@ -0,0 +1,79 @@
+import NotesTable, { NoteTableView } from '@/components/NotesTable';
+import { createClient } from '@/utils/supabase/server';
+import Link from 'next/link';
+import { redirect } from 'next/navigation';
+import { PiCaretLeft } from 'react-icons/pi';
+
+export default async function Notes({
+  searchParams,
+}: {
+  searchParams: { search?: string; tag?: string };
+}) {
+  const supabase = createClient();
+
+  const {
+    data: { user },
+  } = await supabase.auth.getUser();
+
+  if (!user) {
+    return redirect('/login');
+  }
```

Comment:
Do we need this now that we have the middleware working?

Reply:
We don't! Don't even need user object here, will nuke it.